### PR TITLE
Add cache for PostgreSQLCommand

### PR DESCRIPTION
**Pull request by TeslaCN**
Related to #10626 

image

### CODE REVIEW

**Review by tristaZero — CHANGES_REQUESTED** [review]
Is there any chance this cache is cleared or lessened?

File: shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/command/query/PostgreSQLCommand.java [hunk 1]
```diff
@@ -51,6 +53,8 @@
     ROLLBACK(RollbackStatement.class),
     SET(SetStatement.class);
     
+    private static final Map<Class<? extends SQLStatement>, Optional<PostgreSQLCommand>> CACHE = new ConcurrentHashMap<>(16, 1);
```
Comment (taojintianxia) [inline shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/command/query/PostgreSQLCommand.java:56]: maybe XXX_CACHE is better ?

**Comment by TeslaCN** [comment]
> Is there any chance this cache is cleared or lessened?

This is a high frequency method and the elements are few. There is no need to clear.

File: shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/command/query/PostgreSQLCommand.java (hunk 1 above)
Comment (TeslaCN) [reply shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/command/query/PostgreSQLCommand.java]: @tristaZero @taojintianxia Rename to `COMPUTED_CLASSES` may be clearer.

**Review by tristaZero — APPROVED** [review]

**Pull request merged**